classes/genre.rb

```diff
@@ -0,0 +1,20 @@
+require_relative 'music_album'
+
+class Genre
+  attr_accessor :name
+
+  def initialize(name)
+    @id = Random.rand(1...1000)
+    @name = name
+    @items = []
+  end
+
+  def add_item(album)
+    @items << album
+    album.genre = self
+  end
+
+  private
+
+  attr_reader :id, :items
```

Comment:
Since items are used in the add_item method it is better to change the attr_reader to attr_accessor

Reply:
alright will do